sdk/android/src/jni/pc/frame_cryptor_key_manager.h

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2022 LiveKit
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef SDK_ANDROID_SRC_JNI_PC_FRAME_CRYPTOR_KEY_MANAGER_H_
+#define SDK_ANDROID_SRC_JNI_PC_FRAME_CRYPTOR_KEY_MANAGER_H_
+
+#include <jni.h>
+
+#include "api/crypto/frame_crypto_transformer.h"
+#include "sdk/android/native_api/jni/scoped_java_ref.h"
+
+namespace webrtc {
+namespace jni {
+
+class DefaultKeyManagerImpl : public webrtc::KeyManager {
+ public:
+  DefaultKeyManagerImpl() = default;
+  ~DefaultKeyManagerImpl() override = default;
+
+  /// Set the key at the given index.
+  bool SetKey(const std::string participant_id,
+              int index,
+              std::vector<uint8_t> key) {
+    if (index > webrtc::KeyManager::kMaxKeySize) {
```

Comment:
Is the the key size or the number of keys that should be limited to 32?

Reply:
the index should not be limited, but key bytes need to be limited to 32
https://github.com/webrtc-sdk/webrtc/pull/54/files/b6c80e4a573e216c4ad22652dd76c108d6780737#diff-7c5d6536fc3fee31501ca8f2f430869b0cac2a050abefe6b5077af-R6077af1d92ae,

![image](https://user-images.githubusercontent.com/17733746/219571243-f6e030d8-d2f5-4ef5-99aa-a0d20421bf2e.png)

So there should be some updates here